book/04-git-server/sections/git-daemon.asc

```diff
@@ -50,6 +50,8 @@ You might have noticed that Git daemon is started here with `git` as both group
 
 Modify it to fit your needs and make sure provided user exists on the system.
 
+Also, make sure that the Git binary is indeed located at `/usr/bin/git` and change the path if necessary.
+
```

Comment:
Let's make this part of the paragraph just before it, I think it'll flow better that way.

Reply:
I wasn't sure about the formatting rules here. Since the two user-related sentences are split across separate paragraphs, it felt odd to attach something very different to one of the two.

Reply:
🤔 

> You might have noticed that Git daemon is started here with git as both group and user.
> 
> Modify it to fit your needs and make sure provided user exists on the system.
> 
> Also, make sure that the Git binary is indeed located at /usr/bin/git and change the path if necessary.

You know what? Let's make all of these the same paragraph. Oh, and can you add the word "the" between "make sure" and "provided user"?